cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -128,6 +128,9 @@ func TestPolicyApplied(t *testing.T) {
 			// Reset color that we disable on client when we request --no-color
 			color.NoColor = false
 
+			// Replace non-Windows friendly characters in the test name
+			name = strings.Replace(name, `\`, "_", -1)
```

Comment:
I suggest you place that in testutils, which replacing ":", "\" and others in chain with a function name like normalize_golden() or something around that, (and only call it for the golden filename, don’t replace the name itself)